src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -39,7 +39,7 @@ public EnergyBalanceInfo GetEnergyBalanceForSpecies(MicrobeSpecies species, Biom
 
             // Auto-evo uses the average values of compound during the course of a simulated day
             cached = ProcessSystem.ComputeEnergyBalance(species.Organelles, biomeConditions, species.MembraneType,
-                species.PlayerSpecies, worldSettings, CompoundAmountType.Average);
+                species.PlayerSpecies, species.IsBacteria, worldSettings, CompoundAmountType.Average);
```

Comment:
Hang on, you added the parameter called "hasNucleus" but here you pass in the variable `IsBacteria` should this bool not be flipped here?

Reply:
Since simulationcache pulls it's info from species information, I assumed it would still use IsBacteria since that is used often in Species files. You're right that it should probably be flipped though.